src/Tests/TemplateTests.cs

```diff
@@ -111,7 +111,10 @@ public async Task NServiceBusEndpointPersistence(string persistence)
     }
 
     [Test]
-    public async Task Handler()
+    [TestCase("Default")]
+    // Tests for ImplicitUsings not available
+    [TestCase("net48")]
```

Comment:
Given that that `ImplicitUsings` condition is : `"value": "(framework != \"net472\" && framework != \"net48\")"`, should we be testing more cases here?

Reply:
I only wanted to test the two conditions. That the frameworks chosen result in different ImplicitUsings outcomes are verified by other tests.